src/solver/conv_MP_bidirectional_winograd.cpp

```diff
@@ -201,8 +201,6 @@ bool ConvMPBidirectWinograd<WinoDataH, WinoFilterH, WinoDataW, WinoFilterW>::IsA
         return false;
     }
 
-    assert(params.weights_layout.length() == 0); // _weights_layout is not supported yet
```

Comment:
Sounds reasonable. Though I'd want more clarification in this check: In the original code, why is only `weights_layout` checked here (instead of three)?

I'm asking this because most probably `params.IsLayoutDefault()` will check all three layout on whether they are default, then it becomes an un-equivalent code change compared to original one.

Reply:
> why is only weights_layout checked here (instead of three)?

Because one line of useless code (comment) is enough to remind ;)

Reply:
> then it becomes an un-equivalent code change compared to original one.

Original code does not check anything ;)